.github/workflows/heroku_deploy.yml

```diff
@@ -42,3 +43,17 @@ jobs:
           heroku maintenance:off --app=${{ needs.Setup.outputs.APP_NAME }}
         env:
           HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
+      - name: Update Heroku on what type of app to build next time (Vue or React)
+        run: |
+          config_file=$(./scripts/vue_or_react.sh)
+          current_config=$(heroku config:get BUILDPACK_RUN --app=${{ needs.Setup.outputs.APP_NAME }})
+          if echo "$current_config" | grep -q "config_file"; then
+            echo "Keeping current config of $config_file"
+          else
+            echo "Switching Heroku to build with $config_file instead"
+            heroku plugins:install heroku-builds
+            heroku builds:cache:purge --app=${{ needs.Setup.outputs.APP_NAME }} --confirm=${{ needs.Setup.outputs.APP_NAME }}
+            heroku config:set BUILDPACK_RUN="pipenv run cookiecutter . --config-file $config_file --no-input -f" --app=${{ needs.Setup.outputs.APP_NAME }}
+          fi
+        env:
+          HEROKU_API_KEY: ${{ secrets.HEROKU_API_KEY }}
```

Comment:
`BUILDPACK_RUN` will default to React.
Should we detect a change in the code the dev is working on (i.e. Vue instead of React) we'll make a couple of calls here to tell Heroku to build a Vue app instead. This is done last as we currently have no way to sneak this in before Heroku starts building. So Heroku will build the wrong app and Cypress will likely fail. But the next push of code will work as expected.